dnf-json

```diff
@@ -182,6 +216,22 @@ class Solver():
 
 class DnfJsonRequestHandler(BaseHTTPRequestHandler):
 
+    cache_folders = []
```

Comment:
An instance of `DnfJsonRequestHandler` is created per a request, right? If it isn't, this variable is IMHO not thread-safe.

Reply:
Yes that's right. But that's the forking library that is taking care of this.